values.sample.yaml

```diff
@@ -25,11 +25,36 @@ enableLogCollection: false
 # This value will be provided to you by Tonic and will allow you to authenticate against our private docker repository.
 dockerConfigAuth: <docker-config-auth>
 
+# service account for tonic
+serviceAccount:
+  create: true
+  annotations: {}
+
 # You can set this to a specific Tonic version number if you wish to ensure you always get the same version. Otherwise you will always deploy the latest version of Tonic.
 tonicVersion: latest
 
+# container repository with tonic images. If you don't use the default one you will have to copy images over before deployment
+tonicai:
+  env: {}
+  envRaw: {}
+  notifications:
+    image: quay.io/tonicai/tonic_notifications
+  pii_detection:
+    image: quay.io/tonicai/tonic_pii_detection
+  spark_helper:
+    image: quay.io/tonicai/tonic_spark_helper
+  web_server:
+    image: quay.io/tonicai/tonic_web_server
+    annotations:
+      # This annotation is only required if you are creating an internal facing ELB. Remove this annotation to create public facing ELB.
+      service.beta.kubernetes.io/aws-load-balancer-internal: "true"
+      service.beta.kubernetes.io/azure-load-balancer-internal: "true"
+  worker:
+    image: quay.io/tonicai/tonic_worker
+
 # Use a Tonic license to enable different enterprise features. License keys can be obtained from Tonic.
-# tonicLicense: <license-key>
+# tonicLicenseRaw: <license-key>
+# tonicLicenseSecret: <kubernetes-secret-with-key-tonic-license>
```

Comment:
The license doesn't contain any sensitive information so I'm not sure if it needs to be a secret. However if we are going to make it one, can you make it always a secret similar to how the database password is done?

Reply:
If you're sure it doesn't need a secret it's simpler to leave it plain text 🤷